### ISSUE TITLE
Flutter 3 not compiling

### ISSUE THREAD

**Issue by Faiizii** [flutter-stripe/flutter_stripe#726]
**Describe the bug**
Flutter project is not compiling after upgraded to Flutter 3. See the following logs
````Launching lib\main.dart on N9517 in debug mode...
Running Gradle task 'assembleDebug'...
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stripe-2.5.0/lib/src/widgets/card_field.dart:340:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {
                     ^
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stripe-2.5.0/lib/src/widgets/card_field.dart:497:26: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/flutter/packages/flutter/lib/src/widgets/binding.dart').
          WidgetsBinding.instance!.focusManager.primaryFocus !=
                         ^
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stripe-2.5.0/lib/src/widgets/card_form_field.dart:256:22: Warning: Operand of null-aware operation '?.' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {
                     ^
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stripe-2.5.0/lib/src/widgets/card_form_field.dart:410:26: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/flutter/packages/flutter/lib/src/widgets/binding.dart').
          WidgetsBinding.instance!.focusManager.primaryFocus !=
                         ^
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeAubecsDebitPlatformViewFactory.kt: (13, 1): Class 'StripeAubecsDebitPlatformViewFactory' is not abstract and does not implement abstract base class member public abstract fun create(p0: Context?, p1: Int, p2: Any?): PlatformView defined in io.flutter.plugin.platform.PlatformViewFactory
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeAubecsDebitPlatformViewFactory.kt: (19, 5): 'create' overrides nothing
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformViewFactory.kt: (12, 1): Class 'StripeSdkCardFormPlatformViewFactory' is not abstract and does not implement abstract base class member public abstract fun create(p0: Context?, p1: Int, p2: Any?): PlatformView defined in io.flutter.plugin.platform.PlatformViewFactory
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformViewFactory.kt: (18, 5): 'create' overrides nothing
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformViewFactory.kt: (12, 1): Class 'StripeSdkCardPlatformViewFactory' is not abstract and does not implement abstract base class member public abstract fun create(p0: Context?, p1: Int, p2: Any?): PlatformView defined in io.flutter.plugin.platform.PlatformViewFactory
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformViewFactory.kt: (18, 5): 'create' overrides nothing
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkGooglePayButtonPlatformViewFactory.kt: (12, 1): Class 'StripeSdkGooglePayButtonPlatformViewFactory' is not abstract and does not implement abstract base class member public abstract fun create(p0: Context?, p1: Int, p2: Any?): PlatformView defined in io.flutter.plugin.platform.PlatformViewFactory
e: C:\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-2.5.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkGooglePayButtonPlatformViewFactory.kt: (18, 5): 'create' overrides nothing

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe_android:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 58s
Exception: Gradle task assembleDebug failed with exit code 1
````
![image](https://user-images.githubusercontent.com/41893474/168444013-848ddec2-766e-4ea6-b683-f491ba962149.png)

**Comment by remonh87** [comment]
duplicate of #718  we shipped version 3.0.0 that should solve it

**Comment by Faiizii** [comment]
Thanks for the fix
